packages/eos-components/src/components/InstallContentModal.vue

```diff
@@ -1,80 +1,62 @@
 <template>
 
-  <div>
+  <div v-if="collection">
     <transition name="fade">
       <b-button
-        v-if="!isModalVisible && downloading"
+        v-if="!visible && downloading"
         variant="primary"
         class="install-content rounded-circle"
-        @click="showModal"
+        @click="$emit('showModal')"
       >
         <div :style="buttonStyles" class="button-progress rounded-circle"></div>
-        <DownloadIcon />
+        <ProgressDownloadIcon />
       </b-button>
     </transition>
 
     <b-modal
       id="install-content-modal"
-      v-model="isModalVisible"
-      headerCloseVariant="light"
-      :hideFooter="downloading"
-      @hide="onHide"
+      size="xl"
+      centered
+      :visible="visible"
+      :hideFooter="true"
+      @hide="$emit('hide')"
     >
-      <template #modal-title>
-        Welcome to Endless Learning
-      </template>
-      <h2 v-if="downloading">
-        Downloading&hellip;
-      </h2>
-      <h2 v-else>
-        Download Endless Learning Collection
-      </h2>
-      <p v-if="downloading">
-        You can close this window during the download and continue to
-        explore the existing content.
-      </p>
-
-      <div v-if="downloading" class="d-block text-center">
-        <b-progress
-          :value="progress"
-          :max="100"
-          showProgress
-          animated
-          class="my-2"
-        />
-        <b-button v-b-toggle.details variant="link">
-          More details
-        </b-button>
-        <b-collapse id="details" class="mt-2">
-          <div v-for="job in filteredJobs" :key="job.id">
-            <b-row>
-              <b-col>{{ job.channel_name }}</b-col>
-              <b-col cols="9">
-                <div v-if="job.status === 'COMPLETED'">
-                  Completed
-                </div>
-                <b-progress
-                  v-else
-                  :value="job.percentage"
-                  :max="1"
-                  variant="success"
-                  showProgress
-                  animated
-                />
-              </b-col>
-            </b-row>
-          </div>
-        </b-collapse>
-      </div>
-      <template #modal-footer>
+      <b-container>
+        <h1 class="text-primary">
+          Downloading&hellip;
+        </h1>
+        <h5 class="text-muted">
+          You can close this window during meanwhile
```

Comment:
"during meanwhile" doesn't seem correct English to me. But I understand that this is placeholder text, we'll need a native speaker for the final edits.